cobalt/testing/BUILD.gn

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//testing/test.gni")
+
+test("cobalt_unittests") {
```

Comment:
I would move this whole target up to `//cobalt/BUILD.gn`.

Reply:
I'd consider //cobalt/testing to be a good directory for hosting all tests including unit tests, @andrewsavage1 also recommends this directory.

Reply:
Hmmkay, we can move it later I guess.

Reply:
I would follow the example of base_unittests—rename it `//cobalt/test` (not `//cobalt/testing`) and consider putting cobalt_unittests target in `//cobalt/BUILD.gn` while much of the code it tests can live in `//cobalt/test`

Reply:
Renamed //cobalt/testing to //cobalt/test, I think we can still keep the user_agent_unittests closer to user_agent source code in the user_agent directory.